Update README.md

jennamk14 opened the pull request:
added link to wiki for issue #61 
Added page to wiki on database https://github.com/Imageomics/kabr-tools/wiki/Data-Management-Tips

jennamk14 commented:
@egrace479 Added these revisions based on feedback from code review 
@zhong-al fyi

zhong-al commented:
@jennamk14

The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.

egrace479 commented:
> @jennamk14
> 
> The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.

I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.

jennamk14 commented:
> > @jennamk14
> > The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.
> 
> I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.

ok thanks for the info! how should be point users from the readme to the wiki?

egrace479 commented:
> > > @jennamk14
> > > The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.
> > 
> > 
> > I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.
> 
> ok thanks for the info! how should be point users from the readme to the wiki?

The current pointers are good; using the full URL works.

egrace479 commented on README.md:
```diff
@@ -48,9 +48,11 @@ Recognition from Drone Videos](https://openaccess.thecvf.com/content/WACV2024W/C
-**Figure 3:** A mini-scene is a sub-image cropped from the drone video footage centered on and surrounding a single animal. Mini-scenes simulate the camera as well-aligned with each individual animal in
-the frame, compensating for the movement of the drone and ignoring everything in the large field of
+**Figure 3:** A mini-scene is a sub-image cropped from the drone video footage centered on and surrounding a single animal. Mini-scenes simulate the camera as well-aligned with each animal in
+the frame, compensating for the drone's movement and ignoring everything in the large field of
 view but the animal’s immediate surroundings. The KABR dataset consists of mini-scenes and their
```
```suggestion
the frame, compensating for the drone's movement by focusing on just the animal and its immediate surroundings. The KABR dataset consists of mini-scenes and their
```
Would it be more appropriate to describe it as "and focusing..." or "by focusing..." I think that focus is at least part of how it compensates for the drone movement?

jennamk14 replied:
yes, i like 'focusing' a lot better here

egrace479 reviewed (approved):
I just made the text change. We should also make sure that the sample images have alt-text (it goes between the brackets "[]"), that can be done with a separate issue.
